_commands/scripting/appendOperator.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+>\>
+-----
+
+'>>' is a shell operator that you can use to append the output in an existing file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+$ echo line 2 >> example.txt
+$ cat example.txt
+line 1
+line 2
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+If the file does not exist, '>>' creates the file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+~~~
+
+The appending operator allows for the appending of standard output and standard error to a file.
+
+Appending stdout to the file is appending the output of a file to the end of the other file.
+
+~~~ bash
+$ python random.py >> output.txt
+Traceback (most recent call last):
+  File "random.py", line 2, in <module>
+    print 3 / 0
+ZeroDivisionError: integer division or modulo by zero
+$ cat output.txt
+Random text
+~~~
+
+Appending stderr to the file appends both the stdout and the error message to the end of the other file.
```

Comment:
This statement isn't entirely correct; that's only because you're redirecting stderr to stdout. Like above, this example isn't completely clear on what's happening. Please use a different program (perhaps the error messages from compiling with gcc?)